src/player.ts

```diff
@@ -208,6 +208,16 @@ export default class Player extends EventEmitter {
     });
   }
 
+  removeRoundBoosterEvents( type?: Operator) {
```

Comment:
it should be `removeRoundBoosterEvents( type?: Operator.Income | undefined) `

Since any other type (Operator.Once, Operator.Pass, etc) are invalid for the purpose of this function

Reply:
done